### DISCUSSION TITLE
Error importing model

### DISCUSSION THREAD

**Discussion by Gregorino** [apache/incubator-mxnet#20056] [General] [answered]
I have trained gluon's I3D model and exported it to:
- model-0000.params
- model-symbol.json

When trying to load the model by: 
   mx.gluon.nn.SymbolBlock.imports(symbols_file_path, ['data'], parameters_file_path, ctx=context)

I get the 
UserWarning: Cannot decide type for the following arguments. Consider providing them as input:
	data: None
  input_sym_arg_type = in_param.infer_type()[0]

and returned model is:
SymbolBlock(
<Symbol dense0_fwd> : 1 -> 1
)

I kindly ask for help on this!
(I am using custom build mxnet 1.8 with cuda 11

**Comment by szha** [comment, marked as answer]
The user warning is probably ok to ignore as when you perform the next forward with the SymbolBlock it will know the data type from the input. Is there any subsequent error that blocks you?

**Reply by Gregorino** [reply]
It is as you wrote. According to warning message and strange output I prematurly concluded something was wrong, although the net is working fine. Thank you for help!
